### PR TITLE
[DRAFT] Add editor menu

### PR DESCRIPTION
**Pull request by dayo09**
This commit adds editor menu for .circle and .cfg files.
- circle-tracer for .circle
- configuration-settins for .cfg

Signed-off-by: dayo09 <dayg502@gmail.com>

### CODE REVIEW

**Review by dayo09 — COMMENTED** [review]
https://github.com/Samsung/ONE-vscode/blob/b307d86ed7a4d7479b63aa5d401b9c923f486390/package.json#L65-L72

(Question for the maintainers) Why is .cfg file's lang id "ini"?

**Comment by seanshpark** [comment]
> (Question for the maintainers) Why is .cfg file's lang id "ini"?

Cause .cfg followed INI format: https://en.wikipedia.org/wiki/INI_file

File: package.json [hunk 1]
```diff
@@ -39,7 +39,8 @@
       {
         "command": "onevscode.configuration-settings",
         "title": "configuration-settings",
-        "category": "ONE"
+        "category": "ONE",
```
Comment (dayo09) [inline package.json:42]: image
By hovering upon the icon, the title text "configuration-settings" appears 
By clicking, one configuration window is opened.

File: package.json [hunk 2]
```diff
@@ -49,7 +50,8 @@
       {
         "command": "onevscode.circle-tracer",
         "title": "circle tracer",
-        "category": "ONE"
+        "category": "ONE",
+        "icon" : "$(play-circle)"
```
Comment (dayo09) [inline package.json:54]: image

By hovering upon the icon, the title text "circle tracer" appears 
By clicking, one circle tracer window is opened.

File: package.json [hunk 3]
```diff
@@ -39,7 +39,8 @@
       {
         "command": "onevscode.configuration-settings",
         "title": "configuration-settings",
-        "category": "ONE"
+        "category": "ONE",
+        "icon" : "$(play-circle)"
```
Comment (dayo09) [inline package.json:43]: image

Icon selected from https://code.visualstudio.com/api/references/icons-in-labels#icon-listing
TODO: add ONE product icon

File: package.json (hunk 1 above)
Comment (dayo09) [reply package.json:42]: TODO: import the cfg file at once

File: package.json (hunk 2 above)
Comment (dayo09) [reply package.json:54]: TODO: open the circle file at once

**Pull request closed without merging**